README.md

```diff
@@ -244,81 +247,98 @@ controls = {"CIS-AWS_v1.2.0_1.16"}
+      "platform": "terraform",
+      "provider": "google",
+      "resource_id": "google_compute_firewall.rule-2",
+      "resource_type": "google_compute_firewall",
+      "rule_description": "VPC firewall rules should not permit unrestricted access from the internet to port 22 (SSH). Removing unfettered connectivity to remote console services, such as SSH, reduces a server's exposure to risk.",
+      "rule_id": "FG_R00379",
+      "rule_message": "",
+      "rule_name": "tf_google_compute_firewall_no_ingress_22",
+      "rule_result": "FAIL",
+      "rule_severity": "High",
+      "rule_summary": "VPC firewall rules should not permit ingress from '0.0.0.0/0' to port 22 (SSH)"
     }
-  ]
+  ],
+  "summary": {
+    "filenames": [
+      "../test_infra/cfn/cfntest2.yaml",
+      "../test_infra/tf/"
+    ],
+    "rule_results": {
+      "FAIL": 1,
+      "PASS": 2
+    },
+    "severities": {
+      "Critical": 0,
+      "High": 1,
+      "Informational": 0,
+      "Low": 0,
+      "Medium": 0,
+      "Unknown": 0
+    }
+  }
 }
 ```
 
 **These are the important bits:**
 
+- Rule Results
 - Summary
-- Controls
-- Rules
-
-### Summary
 
-The `summary` block contains a breakdown of the compliance state of your Terraform files. In the output above, the Terraform violated 2 rules and 2 controls, so the test as a whole failed.
+### Rule Results
 
-### Controls
+Each entry in the `rule_results` block is the result of a Rego rule evaluation on a resource. All `rule_results` across multiple CloudFormation and Terraform files and directories are aggregated into this block. In the example above, the resource `S3Bucket1` configured in the `../test_infra/cfn/cfntest2.yaml` CloudFormation template passed the rule `cfn_s3_block_public_access`, and the resource `google_compute_firewall.rule-2` configured in the `../test_infra/tf/` Terraform directory failed the rule `tf_google_compute_firewall_no_ingress_22`.
 
-Regula shows you compliance results for both controls and rules, in addition to which specific resources failed. Above, in the `controls` block, you can see that the Terraform in the example is noncompliant with `CIS-AWS_v1.2.0_1.22`, and the mapped rules that failed are listed underneath (in this case, `tf_aws_iam_admin_policy`).
-
-### Rules
+### Summary
 
-In the `rules` block further down from `controls`, each rule lists the resources that failed. Above, you'll see that the resource `aws_iam_policy.basically_allow_all` was the one that failed the mapped rule -- as noted by `"valid": false`. In contrast, the `aws_iam_policy.basically_deny_all` resource passed.
+The `summary` block contains a breakdown of the `filenames` (CloudFormation templates, Terraform plan files, Terraform HCL directories) that were evaluated, a count of `rule_results` (PASS, FAIL, or UNKNOWN), and a count of `severities` for failed `rule_results`. In the example above, 3 rule results were evaluated, of which 1 had a `FAIL` result with a `High` severity.
```

Comment:
We don't currently have an `UNKNOWN` value for `rule_result`, but it might make sense to add one. Is this just planning ahead?

The `Unknown` that we currently have is in `severity`, since it's not required that rules specify a severity.

Reply:
Yeah, I would leave out the `UNKNOWN` here for now.

Reply:
ah - good point. removing.